static/css/impala/forms.less

```diff
@@ -246,7 +246,7 @@ textarea {
       font-weight: bold;
       display: block;
     }
-    input[type=text] {
+    input[type=text].api-input {
```

Comment:
instead of putting your rule in `forms.less`, this is a good opportunity to make a new file like `devhub-api.less` or something and put your rule in there like this:
````
input[type=text].api-input {
    ....
}
````
Or better yet, give your [form](https://github.com/Nolski/olympia/blob/signing-api-key-page/apps/devhub/templates/devhub/api/key.html#L13) a class of `api-input` and add styles like:
````
.api-input {
    input[type=text] {
        ...
    }
    // more styles later...
}
````

You can never make too many css files. They all get compressed together. You will also need to add the new file to [the devhub bundle](https://github.com/Nolski/olympia/blob/signing-api-key-page/lib/settings_base.py#L530).